package.json

```
@@ -30,9 +30,10 @@
   },
   "homepage": "https://github.com/axelarnetwork/axelar-cgp-solidity#readme",
   "dependencies": {
-    "@axelar-network/axelar-gmp-sdk-solidity": "5.6.4"
+    "@axelar-network/axelar-gmp-sdk-solidity": "^5.7.0"
```

```suggestion
    "@axelar-network/axelar-gmp-sdk-solidity": "5.7.0"
```
Keep it fixed or use `~5.7.0`? We want to avoid automatically bumping the minor version if someone does `npm install`.

---

It will install what's in the package lock. But I agree that we better lock the dependency.